index.js

```diff
@@ -1,72 +1,137 @@
-    this.books = this.books.filter((book, i) => i !== index);
-    this.updateLocalStorage();
+  static removeBook(book) {
+    let books = Store.getBooks();
+    books = books.filter((currentBook) => !(currentBook.title === book.title
+      && currentBook.author === book.author));
+    localStorage.setItem('books', JSON.stringify(books));
   }
+}
 
-  renderBookList() {
-    this.bookList.innerHTML = '';
-    this.books.forEach((book, index) => {
-      const li = document.createElement('p');
-      li.innerHTML = `"${book.title}" by ${book.author} <br> `;
-      this.bookList.appendChild(li);
-      li.dataset.index = index;
+// UI class
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+    const bookList = document.getElementById('book-list');
+    bookList.innerHTML = '';
 
-      const removeButton = document.createElement('button');
-      removeButton.innerHTML = 'Remove';
-      removeButton.addEventListener('click', () => {
-        const index = parseInt(li.dataset.index, 10);
-        this.removeBook(index);
-        this.renderBookList();
-      });
-      li.appendChild(removeButton);
-      if (index % 2 === 1) {
-        li.classList.add('white');
-      }
+    books.forEach((book) => {
+      const bookElement = document.createElement('div');
+      bookElement.classList.add('book');
+      bookElement.innerHTML = `
+          <div>
+          <span><strong>Title:</strong> ${book.title}</span>
+          <span><strong>Author:</strong> ${book.author}</span>
+          </div>
+          <button class="delete">Remove</button>
+        `;
+      bookList.appendChild(bookElement);
     });
   }
 
-  resetForm() {
-    this.addBookForm.reset();
-  }
+  static addBook() {
+    const titleInput = document.getElementById('titleInput');
+    const authorInput = document.getElementById('authorInput');
+    const title = titleInput.value.trim();
+    const author = authorInput.value.trim();
 
-  updateLocalStorage() {
-    localStorage.setItem('books', JSON.stringify(this.books));
+    const book = new Book(title, author);
+    Store.addBook(book);
+    UI.displayBooks();
+    titleInput.value = '';
+    authorInput.value = '';
   }
 
-  getFromLocalStorage() {
-    const storedBooks = localStorage.getItem('books');
-    if (storedBooks !== null) {
-      this.books = JSON.parse(storedBooks);
+  static removeBook(target) {
+    if (target.classList.contains('delete')) {
+      const bookElement = target.parentElement;
+      const title = bookElement.querySelector('span:first-child').textContent.split(': ')[1];
+      const author = bookElement.querySelector('span:nth-child(2)').textContent.split(': ')[1];
+      const book = new Book(title, author);
+      Store.removeBook(book);
+      UI.displayBooks();
     }
   }
 }
-const bookCollection = new BookCollection();
-bookCollection.resetForm();
\ No newline at end of file
+
+// Event listeners
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
```

Comment:
- [OPTIONAL] Kindly make sure when the page is reloaded, the user can still be able to see the list of books that is currently stored in the localStorage.

  - ### HIGHLIGHTS


  ![chrome-capture-2023-5-10](https://github.com/Jessekay/AwesomeBooks/assets/87197412/70ff5ca7-04bd-41a6-9df6-5b64b8e58777)